src/components/docEditor/data.js

```diff
@@ -0,0 +1,65 @@
+export default {
+  doc: {
+    title: 'Outrage over police injustice is older than the nation',
+    slug: 'outrage-over-police-injustice-is-older-than-the-nation',
+    description: 'A tip of the internet is assumed to be an unlet baritone. The literature would have us believe that a castled anthony is not but a jaw. Few can name a rimless cook that isn',
+    trash: false,
```

Comment:
Right now, documents that have never been in the trash don't have any `trash` property. Our boolean input looks for a specific value for the field. Maybe this field needs special treatment OR we need to start new docs with `trash: false`. CC @boutell